tests/contrib/django/test_django_snapshots.py

```diff
@@ -55,7 +55,7 @@ def daphne_client(django_asgi, additional_env=None):
     client.wait()
 
     try:
-        yield client
+        yield (client, proc)
```

Comment:
should we just create two daphne clients? One that yields (client,proc) and one that yields just client (like the original)? That way you don't have to touch all these other tests because the client is returning something we don't care about in most cases?
E.g.
```
def daphne_client_1
    yield client, proc

def daphne_client_2()
    client, proc = daphne_client_1()
    yield client
```

Reply:
I think I'm gonna keep it as is for now, since I'm already yielding both anyways (and I can't really separate them). It does feel a little gross that I'm touching those other tests, but I figured I could clean this up later if it becomes excessive.